src/Modal.js

```diff
@@ -472,7 +475,7 @@ class Modal extends React.Component {
     let autoFocus = this.props.autoFocus;
     let modalContent = this.getDialogElement();
     let current = activeElement(ownerDocument(this));
-    let focusInModal = current && contains(modalContent, current);
+    let focusInModal = modalContent && current && contains(modalContent, current);
```

Comment:
see https://github.com/react-bootstrap/react-bootstrap/issues/2726#issuecomment-329047753

i think especially w/react 16 this might be a little flaky, since `modalContent` might not be the actual modal content... or at least not all of it

Reply:
i'm not sure I understand that, don't we control this DOM node?

Reply:
No – it's supposed to be the DOM node corresponding to `children`, but the actual node could be e.g. a comment or something per the linked React-Bootstrap issue if e.g. `children` don't end up rendering anything. Perhaps it could be something even weirder with React 16 – like a text node or something.

Reply:
i'm not sure how to deal with this without supplying our own wrapping div, children should render a single element

Reply:
we'd make a `<RefHolder>` component, attach a `ref` to it, then `findDOMNode` it... but this isn't really a regression or anything, so i guess this is fine for now